tests/formats/epoch_processing/README.md

```diff
@@ -50,3 +58,15 @@ Sub-transitions:
 - `pending_deposits` (>=Electra)
 
 The resulting state should match the expected `post` state.
+
+## Condition (alternative)
+
+Instead of having a different handler for each sub-transition, a single handler for all cases, should load `pre_full` state, call `process_epoch` and then assert that the result state should match `post_full` state.
```

Comment:
```suggestion
Instead of having a different handler for each sub-transition, a single handler for all cases should load `pre_full` state, call `process_epoch` and then assert that the result state should match `post_full` state.
```